CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Thank You
+
+We are grateful that you considered contributing to metaKGP-wiki. Please follow the below guidelines for contribution.
+
+Follow the golder rule
+
+> Code, Document, Raise Issues and Review like you have to explain everything to a new developer without saying a word.
+
+## First Time
+
+The first thing to do always is to run the program and note how it works. One of the best ways to do this is to launch the application and go through the files that it launches in the sequential manner.
+
+Also take a note of the coding style and dependencies that the project follows.
+
+## Choosing issues
+
+It is recommended to go through the issues tab once you have ran the program once and look for easy issues.
+
+If an issue is not assigned; follow through the comments, ask queries about it and then start working on it.
+
+## Creating issues
+
+If you face any reproducable problem while launching the game or any feature you think should be there, file an issue in the github repository.
```

Comment:
launching the docker containers *

Reply:
Add a note here that they should try to figure it out on their own before opening an issue on this repo. 🙂